src/core/public/fatal_errors/fatal_errors_service.tsx

```diff
@@ -123,4 +125,17 @@ export class FatalErrorsService {
       container
     );
   }
+
+  private setupGlobalErrorHandlers(fatalErrorsSetup: FatalErrorsSetup) {
+    if (window.addEventListener) {
+      window.addEventListener('unhandledrejection', function(e) {
+        console.log(`Detected an unhandled Promise rejection.\n${e.reason}`); // eslint-disable-line no-console
+      });
+    }
+
+    window.onerror = (err, url, line) => {
```

Comment:
minor, but `err: string | Error`, and in case of `Error`string here https://github.com/elastic/kibana/pull/43610/files#diff-fc5483a74e254d59cb93c1d47eb04f3dR137 will have value `Error: Error: ${message} (${url}:${line})`